packages/web-components/src/divider/divider.styles.ts

```diff
@@ -0,0 +1,125 @@
+import { css } from '@microsoft/fast-element';
+import {
+  colorBrandForeground1,
+  colorBrandStroke1,
+  colorNeutralForeground1,
+  colorNeutralForeground2,
+  colorNeutralForeground3,
+  colorNeutralStroke1,
+  colorNeutralStroke2,
+  colorNeutralStroke3,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontWeightRegular,
+  strokeWidthThin,
+} from '../theme/design-tokens.js';
+
+/** Divider styles
+ * @public
+ */
+export const styles = css`
+  :host {
+    display: flex;
+  }
+
```

Comment:
I think we want to use the `${display("flex")}` (display) helper here from foundation.